pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+## Associated issues
+
```

Comment:
What do you think about adding a section to link the the repo that contains the scripts the dag runs? I can't think of a good name but something like

## Associated Repo / Companion Repo / Code Repo

Reply:
I think that this would be a great reminder to point reviewers to the code that a DAG runs! 🙏